src/realm/exec/realm_trawler.cpp

```diff
@@ -355,6 +368,18 @@ class Group : public Array {
         }
         return "Unknown";
     }
+    std::vector<realm::BinaryData> get_changesets()
```

Comment:
this can be `const`

Reply:
Probably - but not a big deal. I will change this if I have to make other changes.